### PR TITLE
ticdc add output old value config

### PR DESCRIPTION
**Pull request by sdojjy**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-changefeed-config.md [hunk 1]
```diff
@@ -232,6 +232,14 @@ enable-partition-separator = true
 # 默认值为 "json"。
 # encoding-format = "json"
 
+[sink.open]
+# 是否输出行数据更改前的值。默认值为 true。关闭后，Update 事件不会输出 "p" 字段的数据。
+# output-old-value = false
```
Comment (Oreoxmt) [inline ticdc/ticdc-changefeed-config.md:237]:
```suggestion
# output-old-value = true
```
建议示例中使用默认值

File: ticdc/ticdc-changefeed-config.md [hunk 2]
```diff
@@ -232,6 +232,14 @@ enable-partition-separator = true
...
+
+[sink.debezium]
+# 是否输出行数据更改前的值。默认值为 true。关闭后，Update 事件不会输出 "before" 字段的数据。
+# output-old-value = false
```
Comment (Oreoxmt) [inline ticdc/ticdc-changefeed-config.md:241]:
```suggestion
# output-old-value = true
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: ticdc/ticdc-open-api-v2.md [hunk 3]
```diff
@@ -301,19 +301,21 @@ curl -X GET http://127.0.0.1:8300/api/v2/health
...
+| `protocol`                    | `STRING` 类型，对于 MQ 类的 Sink，可以指定消息的协议格式。目前支持以下协议：`canal-json`、`open-protocol`、`avro` 和 `maxwell`。    |
+| `schema_registry`             | `STRING` 类型，schema registry 地址。（非必选）                                                               |
+| `terminator`                  | `STRING` 类型，换行符，用来分隔两个数据变更事件。默认值为空，表示使用 `"\r\n"` 作为换行符。（非必选）                                       |
+| `transaction_atomicity`       | `STRING` 类型，事务一致性等级。（非必选）                                                                          |
+| `only_output_updated_columns` | `BOOLEAN` 类型，对于 MQ 类型的 Sink 中的 `canal-json` 和 `open-protocol`，表示是否只向下游同步有内容更新的列。默认值为 `false`。（非必选） |
+| `cloud_storage_config`        | storage sink 配置。（非必选）                                                                              |
+| `open`                        | open protocol 配置。（非必选）                                                                             |
+| `debezium`                    | debezium sink 配置。（非必选）                                                                             |
```
Comment (Oreoxmt) [inline ticdc/ticdc-open-api-v2.md:318]:
```suggestion
| `open`                        | Open Protocol 配置。（非必选）                                                                             |
| `debezium`                    | Debezium Protocol 配置。（非必选）                                                                             |
```
"TiCDC Debezium Protocol": https://docs.pingcap.com/zh/tidb/dev/ticdc-debezium

**Review by flowbehappy — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

File: ticdc/ticdc-open-api-v2.md [hunk 4]
```diff
@@ -358,6 +360,18 @@ curl -X GET http://127.0.0.1:8300/api/v2/health
 | `file_cleanup_cron_spec`   | `STRING` 类型，定时清理任务的运行周期，与 crontab 配置兼容，格式为 `<Second> <Minute> <Hour> <Day of the month> <Month> <Day of the week (Optional)>`。|
 | `flush_concurrency`   | `INT` 类型，上传单个文件的并发数。|
 
+`sink.open` 参数说明如下：
+
+| 参数名                | 说明                                                            |
+|:-------------------|:--------------------------------------------------------------|
+| `output_out_value` | `BOOLEAN` 类型，是否输出行数据更改前的值。默认值为 `true`。关闭后，Update 事件不会输出 "p" 字段的数据。 |
```
Comment (qiancai) [inline ticdc/ticdc-open-api-v2.md:367]:
```suggestion
| `output_old_value` | `BOOLEAN` 类型，是否输出行数据更改前的值。默认值为 `true`。关闭后，Update 事件不会输出 "p" 字段的数据。 |
```

File: ticdc/ticdc-open-api-v2.md [hunk 5]
```diff
@@ -358,6 +360,18 @@ curl -X GET http://127.0.0.1:8300/api/v2/health
...
+
+`sink.debezium` 参数说明如下：
+
+| 参数名                | 说明                                                                 |
+|:-------------------|:-------------------------------------------------------------------|
+| `output_out_value` | `BOOLEAN` 类型，是否输出行数据更改前的值。默认值为 `true`。关闭后，Update 事件不会输出 "before" 字段的数据。 |
```
Comment (qiancai) [inline ticdc/ticdc-open-api-v2.md:373]:
```suggestion
| `output_old_value` | `BOOLEAN` 类型，是否输出行数据更改前的值。默认值为 `true`。关闭后，Update 事件不会输出 "before" 字段的数据。 |
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**